### PR TITLE
New Parameter for Commit Message Length - #648

### PR DESCRIPTION
**Pull request by bigld**
Two setting parameter to adjust commit comment length (50-100) and Body line size (72-100)

### CODE REVIEW

**Comment by larshp** [comment]
issue https://github.com/larshp/abapGit/issues/648

File: src/zabapgit_persistence.prog.abap [hunk 1]
```diff
@@ -1534,6 +1534,11 @@ ENDCLASS.
 CLASS lcl_settings DEFINITION FINAL.
 
   PUBLIC SECTION.
+    CONSTANTS: c_commitmsg_comment_length_dft TYPE i VALUE 50.
+    CONSTANTS: c_commitmsg_comment_length_max TYPE i VALUE 100.
+    CONSTANTS: c_commitmsg_body_size_dft      TYPE i VALUE 72.
+    CONSTANTS: c_commitmsg_body_size_max      TYPE i VALUE 100.
```
Comment (larshp) [inline src/zabapgit_persistence.prog.abap]: do we need the max values, just let the users decide instead?

Comment (bigld) [inline src/zabapgit_persistence.prog.abap]: tried to keep the values near the recommendation, but after some test it seems no one checks or even care about this
I will change this, so it´s up to the user to find suitable max values

**Comment by larshp** [comment]
@bigld when do you think you will have time to update the PR, removing the max values?

**Comment by bigld** [comment]
@larshp max length is removed - PR updated

**Pull request merged**